src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -23,7 +25,7 @@ public sealed class ReadOnlyCollectionBuilder<T> : IList<T>, System.Collections.
         private int _version;
 
         [NonSerialized]
-        private Object _syncRoot;
+        private object _syncRoot;
```

Comment:
Ok.  That's not what's commonly done, e.g.
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Collections/Generic/List.cs#L163
but I'm fine with it.